### PR TITLE
Config fixes for Ubuntu 14.04

### PR DESCRIPTION
**Pull request by Hwesta**
Apache now expects sites-[available|enabled] to end in .conf, so update dev creation to create 000-default.conf instead of [000-]default.

Gearman no longer creates a PID file that can be checked for. Instead, check the status directly by asking gearadmin for the PID.

refs #7293

### CODE REVIEW

File: src/MCPClient/init/archivematica-mcp-client.conf [hunk 1]
```diff
@@ -34,11 +34,20 @@ pre-start script
     # Check that $CONF directory exists
     [ -d $CONF ]
 
-    # Wait for Gearman service
-    while [ ! -f /var/run/gearman/gearmand.pid ]
+    # Wait for Geraman to start - timeout eventually
```
Comment (mistydemeo) [inline src/MCPClient/init/archivematica-mcp-client.conf]: Typo!

File: src/MCPClient/init/archivematica-mcp-client.conf [hunk 2]
```diff
@@ -34,11 +34,20 @@ pre-start script
...
+    for i in $(seq 1 10)
     do
-        sleep 20
+        if [ "`gearadmin --getpid`" != '' ]; then
```
Comment (mistydemeo) [inline src/MCPClient/init/archivematica-mcp-client.conf]: Check `$?` rather than length of string, maybe?

Comment (mistydemeo) [inline src/MCPClient/init/archivematica-mcp-client.conf]: Or structure as

``` sh
for i in $(seq 1 10)
do
    gearadmin --getpid && break
    sleep 3
done
```

File: src/MCPClient/init/archivematica-mcp-client.conf [hunk 3]
```diff
@@ -34,11 +34,20 @@ pre-start script
...
+            break
+        else
+            sleep 3
+        fi
     done
+    if [ "`gearadmin --getpid`" != '' ]; then
```
Comment (mistydemeo) [inline src/MCPClient/init/archivematica-mcp-client.conf]: Similarly, can be shortened to:

``` sh
gearadmin --getpid || exit 1
```

File: src/MCPClient/init/archivematica-mcp-client.conf (hunk 1 above)
Comment (Hwesta) [inline src/MCPClient/init/archivematica-mcp-client.conf]: Haha fixed.

File: src/MCPClient/init/archivematica-mcp-client.conf (hunk 3 above)
Comment (Hwesta) [inline src/MCPClient/init/archivematica-mcp-client.conf]: That's a much better way of doing it. Updated for MCPClient and MCPServer.

**Comment by mistydemeo** [comment]
:+1:

**Pull request merged**